client/src/components/Visualization/Visualization.css

```diff
@@ -0,0 +1,21 @@
+.visualization-container {
+  width: 100%;
+}
+
+.chart {
+  margin-bottom: 50px;
+}
+
+.margin-right {
+  margin-right: 50px;
+}
+
+.chart-container {
+  background-color: #fff;
```

Comment:
Same comments about color in this file.  You want to keep your colors consistent cross-file so you have a site identity and can quickly change them.